src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.237.1.mdx

```diff
@@ -0,0 +1,32 @@
+---
+subject: Browser agent
+releaseDate: "2023-08-02"
+version: 1.237.1
+features: []
+bugs: ["SPA feature respects ajax deny_list for fetch","prevent dollar symbol only variable names","Undefined deny list ignores block internal","Fix serialization issue with array data"]
+security: []
+---
+
+## v1.237.1
+
+### Bug Fixes
+
+#### SPA feature respects ajax deny_list for fetch
+The SPA feature will now properly respect the ajax deny list and block internal settings for fetch calls. In prior versions, the SPA feature only consulted the ajax deny list for XMLHttpRequest calls.
+
+#### prevent dollar symbol only variable names
+Prevent the build process of the agent from creating minified variable names that only contain the dollar symbol. If such a variable is immediately followed by an ampersand, as could be the case in a minified logic block, it could be picked up by  string replacement logic applied to outgoing HTML within their server.
+
+#### Undefined deny list ignores block internal
+In cases where the customer's configuration does not include the `init.ajax.deny_list` property, the agent would not properly account for the `init.ajax.block_internal` setting and would not block ajax events for internal agent network calls.
+
+#### Fix serialization issue with array data
+Update the harvester to account for array payloads, which in niche cases could convert array payloads to indexed objects.
+
+## Support statement:
+
+New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Older releases will no longer be supported when they reach [end-of-life](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/). Release dates are reflective of the original publish date of the agent version.
+
+New Browser Agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```

Comment:
```suggestion
Browser agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```